TrailBlaze/View Controllers/HomeViewController.h

```diff
@@ -14,6 +14,7 @@ NS_ASSUME_NONNULL_BEGIN
 @interface HomeViewController : UIViewController 
 @property (strong, nonatomic) MKPolyline *cloudPolyline;
 @property (strong, nonatomic) PFUser *cloudUser;
+@property BOOL isRerun;
```

Comment:
same as my other comment, does this need to be public?

Reply:
The "rerun selection view controller" will set this value to true before performing the segue to the homeViewController that's why I have it public